src/ComplexNets/AdjacencyListGraph.h

```diff
@@ -187,6 +188,26 @@ class AdjacencyListGraph
         return VertexFinder<Vertex, VertexContainer>::find(id, vertices);
     }
 
+    /**
+     * Method: getMinVertexId
+     * ----------------------
+     * Description: obtains the minimum vertex id in the graph.
+     * @return the minimum vertex id in the graph
+     */
+    VertexId getMinVertexId() const
+    {
+        VertexId currMin = UINT_MAX;
+        VerticesConstIterator it = verticesConstIterator();
+        while(!it.end()) {
+            VertexId currId = (*it) -> getVertexId();
+            if(currId < currMin) {
```

Comment:
There are issues with whitespace here and above, why don't you try running scripts/format.sh and committing the results?